_includes/meshery.html

```diff
@@ -207,7 +207,7 @@ <h4>Bash user</h4>
       <div class="highlight highlight-source-shell">
       <div style="width:auto; height:auto; padding:0px; position:relative;">
             <pre><code>
- $ curl -L https://git.io/meshery | bash -    
+ $ curl -L https://git.io/meshery | sudo bash -    
```

Comment:
It's not obvious, but you'll want to add `sudo` to [line 216](https://github.com/layer5io/layer5/pull/286/files#diff-a25801561d5b4546bebda5bdc902bf53L216) as well. This line defines what gets copied into the user's clipboard when clicking the copy button.

Reply:
Okay @leecalcote,  I completely missed this one. Thanks I'll update it now.